### ISSUE TITLE
Org Stats charts are not usable due to spike protection threshold UX

### ISSUE THREAD

**Issue by Dhrumil-Sentry** [getsentry/sentry#64337]
### Problem Statement

It's very difficult to understand whats going on in this chart:- User can toggle the threshold off but we likely need some more UX love here

![image](https://github.com/getsentry/sentry/assets/109983869/74fd8637-4ec3-4661-b52b-4f3e6aba2b9a)

### Product Area

Stats

**Comment by mwarkentin** [comment]
Log y-axis is another way to "compress" these types of graphs that can be hard to parse when rendered with linear scale.

**Comment by realkosty** [comment]
Equivalent of this issue in Tags heatmap:
<img width="1082" alt="Screenshot 2023-12-08 at 2 15 38 PM" src="https://github.com/getsentry/sentry/assets/490201/bb572c82-fe85-4c6a-8463-f14dd3a60291">
